image/v1/types.go

```diff
@@ -22,6 +22,12 @@ type ImageList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // Image is an immutable representation of a container image and metadata at a point in time.
+// Images are named by taking a hash of their contents (metadata and content) and any change
+// in format, content, or metadata results in a new name. The images resource is primarily
+// for use by cluster administrators and integrators - end users instead access images via the
+// imagestreamtags or imagestreamimages resources. While image metadata is stored in the API,
+// any integration that accepts images must implement its own storage and use the image
+// resource to manage referential integrity with image streams.
```

Comment:
i'm not clear on what this last sentence is saying.   s/accepts/manages/ ?  

implements its own storage for what?  the metadata, or for the image content itself?  i would assume the latter.

why does referential integrity to the imagestreams matter?  am i not allowed to just create images(image metadata) without updating the imagestreams?  why not?